### ISSUE TITLE
bad conversion of vars after usage

### ISSUE THREAD

**Issue by graingert** [decaffeinate/decaffeinate#203]
```coffee
getComputedStyle = @getComputedStyle or \
  (el, pseudo) ->
    @getPropertyValue = (prop) ->
      prop = 'styleFloat' if prop is 'float'
      prop.replace(getComputedStyleRX, (_, _char)->
        _char.toUpperCase()
      ) if getComputedStyleRX.test prop
      currentStyle = el.currentStyle
      `(currentStyle != null ? currentStyle[prop] : void 0) || null`
    @
getComputedStyleRX = /(\-([a-z]){1})/g
```

becomes

```js
  // getComputedStyle shim, from http://stackoverflow.com/a/21797294
  let getComputedStyle = this.getComputedStyle || 
    function(el, pseudo) {
      this.getPropertyValue = function(prop) {
        if (prop === 'float') { prop = 'styleFloat'; }
        if (getComputedStyleRX.test(prop)) { prop.replace(getComputedStyleRX, (_, _char)=> _char.toUpperCase()
        ); }
        let { currentStyle } = el;
        return (currentStyle != null ? currentStyle[prop] : void 0) || null;
      };
      return this;
    };
  getComputedStyleRX = /(\-([a-z]){1})/g;
```

**Comment by graingert** [comment]
should be:

```
  let getComputedStyleRX = /(\-([a-z]){1})/g;
  // getComputedStyle shim, from http://stackoverflow.com/a/21797294
  let getComputedStyle = this.getComputedStyle || 
    function(el, pseudo) {
      this.getPropertyValue = function(prop) {
        if (prop === 'float') { prop = 'styleFloat'; }
        if (getComputedStyleRX.test(prop)) { prop.replace(getComputedStyleRX, (_, _char)=> _char.toUpperCase()
        ); }
        let { currentStyle } = el;
        return (currentStyle != null ? currentStyle[prop] : void 0) || null;
      };
      return this;
    };
```

**Comment by eventualbuddha** [comment]
This is handled by [add-variable-declarations](https://github.com/eventualbuddha/add-variable-declarations) and is tracked in https://github.com/eventualbuddha/add-variable-declarations/issues/3.

**Comment by eventualbuddha** [comment]
Fixed in v2.9.0.
